Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -37,11 +38,6 @@ varnish
                         - 123.123.123.2
                     base_url: yourwebsite.com
 
-``tags_header``
-"""""""""""""""
-
-**type**: ``string`` **default**: ``X-Cache-Tags``
-
 Header for sending tag invalidation requests to Varnish.
```

Comment:
you moved the title and info, but not this line. please move it along to the tags_header section.